scripts/nqp/actions/skills.py

```diff
@@ -352,3 +354,72 @@ def _build_cooldown_reduction_effect(self, entity: EntityID) -> ReduceSkillCoold
 
     def get_animation(self, aesthetic: Aesthetic):
         return aesthetic.sprites.attack
+
+
+@data_defined_skill
+class TarAndFeather(Skill):
+    """
+    TarAndFeather skill for an entity
+    """
+    name = "tar_and_feather"
+
+    def __init__(self, user: EntityID, target_tile: Tile, direction: DirectionType):
+        super().__init__(user, target_tile, direction)
+        self.affliction_name = 'flaming'
+        self.affliction_duration = 5
+        self.reduced_modifier = 0.5
+        self.cone_size = 1
+
+    def build_effects(self, hit_entity: EntityID) -> List[Effect]:
+        """
+        Build the skill effects
+        """
+
+        # the cone should start where the hit occurred and in the direction of the projectile.
+        entity_position = world.get_entitys_position(hit_entity)
+        entities_in_cone = world.get_affected_entities(entity_position, Shape.CONE, self.cone_size, self.direction)
+        # we should also ignore the hit entity and the projectile from the extra effects
+        entities_in_cone = [x for x in entities_in_cone if x is not hit_entity and x is not self.projectile]
+
+        reduced_effects = []
+        for entity_in_cone in entities_in_cone:
+            reduced_effects += self._create_effects(target=entity_in_cone, modifier=self.reduced_modifier)
+            logging.warning(f"creating effects for {entity_in_cone}")
+
+        first_hit_effects = self._create_effects(target=hit_entity, success_effects=reduced_effects)
```

Comment:
Chain this "reduced" effects to the first hit effects so they are only triggered if the projectile hits.